### PR TITLE
Make cli and rest consistent

### PR DESCRIPTION
**Pull request by orkunkl**
Resolve https://github.com/iov-one/iovns/issues/230

### CODE REVIEW

File: swagger-ui/swagger-ui/swagger.yaml [hunk 1]
```diff
@@ -2044,7 +2044,7 @@ paths:
           description: domain or owner not found
         "500":
           description: Internal Server Error
-  "/domain/query/resolveAccount":
+  "/starname/query/resolve":
```
Comment (davepuchyr) [inline swagger-ui/swagger-ui/swagger.yaml:2047]: `curl -X POST "http://localhost:1318/starname/query/resolve" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"domain\":\"iov\",\"name\":\"dave\"}"` 

yields

```
{
  "error": "unknown request: resolve"
}
```

with

```
[dave@debt iovns]$ iovnscli version --long | jq
{
  "name": "iovns",
  "server_name": "iovnsd",
  "client_name": "iovnscli",
  "version": "0.4.3-5-g24f12cb",
  "commit": "24f12cbc9d150acf11a32c915f216b5d231c3f8a",
  "build_tags": "",
  "go": "go version go1.14.4 linux/amd64"
}
```

Am I missing something?

Comment (davepuchyr) [reply swagger-ui/swagger-ui/swagger.yaml:2047]: Same story with `domainInfo`
```
{
  "error": "unknown request: domainInfo"
}
```
The other endpoints are working. :)

Comment (orkunkl) [reply swagger-ui/swagger-ui/swagger.yaml:2047]: If you are running iovnscli rest server locally pointing to galaxynet, that error is expected. Try to run iovnsd locally and point iovnscli rest to the local instance

Comment (davepuchyr) [reply swagger-ui/swagger-ui/swagger.yaml:2047]: I was/am running `iovnsd` locally.  
```
{
  "node_info": {
    "protocol_version": {
      "p2p": "7",
      "block": "10",
      "app": "0"
    },
    "id": "1085ee50f7a5f78a2690addd8be538755bb9a632",
    "listen_addr": "tcp://0.0.0.0:46656",
    "network": "iovns-galaxynet",
    "version": "0.33.3",
    "channels": "4020212223303800",
    "moniker": "iovns-galaxynet",
    "other": {
      "tx_index": "on",
      "rpc_address": "tcp://0.0.0.0:46657"
    }
  },
  "application_version": {
    "name": "iovns",
    "server_name": "iovnsd",
    "client_name": "iovnscli",
    "version": "0.4.3-5-g24f12cb",
    "commit": "24f12cbc9d150acf11a32c915f216b5d231c3f8a",
    "build_tags": "",
    "go": "go version go1.14.4 linux/amd64"
  }
}
```

Comment (davepuchyr) [reply swagger-ui/swagger-ui/swagger.yaml:2047]: My bad, I was pointed at the wrong port.

**Pull request merged**